UPGRADE.md

```diff
@@ -1,5 +1,29 @@
 # Upgrade
 
+## 2.1.0
+
+### FOSRestRouting Bundle removed
```

Comment:
deprecation not removal, see https://github.com/sulu/sulu/blob/2.6/UPGRADE-2.x.md#deprecate-usage-of-fos-rest-routing